packages/esm-appointments-app/src/appointments-metrics/appointments-metrics.component.tsx

```diff
@@ -27,17 +27,20 @@ const AppointmentsMetrics: React.FC = () => {
           label={t('patients', 'Patients')}
           value={totalAppointments}
           headerLabel={t('scheduledAppointments', 'Scheduled appointments')}
+          view="patients"
         />
         <MetricsCard
           label={t(highestServiceLoad?.serviceName)}
           value={highestServiceLoad?.count ?? '--'}
           // FIX: Strange translation string
           headerLabel={t('highestServiceVolume', 'High volume Service.  today')}
```

Comment:
This doesn't affect this PR, but what should this translation string read as? `High volume Service. today` seems a bit off. Or is this just a placeholder?

Reply:
its a placeholder